apps/application-system/api/src/app/modules/payment/payment.model.ts

```diff
@@ -69,8 +69,8 @@ export class Payment extends Model<Application> {
   user4?: string
 
   @Column({
-    type: DataType.JSONB,
-    defaultValue: {},
+    type: DataType.STRING,
+    defaultValue: '',
   })
   @ApiPropertyOptional()
   definition?: string
```

Comment:
Let's revert the changes here and change `string` to `object`.

See [application.model](https://github.com/island-is/island.is/blob/main/apps/application-system/api/src/app/modules/application/application.model.ts#L82) as reference.